testutil/garbage/garbage.go

```diff
@@ -0,0 +1,304 @@
+package garbage
+
+import (
+	"math"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime/datamodel"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+	"github.com/ipld/go-ipld-prime/must"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+	"github.com/multiformats/go-multihash"
+)
+
+type Options struct {
+	initialWeights map[datamodel.Kind]int
+	weights        map[datamodel.Kind]int
+	blockSize      uint64
+}
+
+type generator func(count uint64, opts Options) (uint64, datamodel.Node)
+
+type hasher struct {
+	code   uint64
+	length int
+}
+
+const charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789`~!@#$%^&*()-_=+[]{}|\\:;'\",.<>?/ \t\n☺💩"
+
+var codecs []uint64
+var hashes []hasher
+var kinds datamodel.KindSet
+var generators map[datamodel.Kind]generator
+var runes []rune
```

Comment:
nit: you can group these with `var ( [many lines] )`, which will also give you nice alignment